src/NHSD.BuyingCatalogue.Ordering.Api/appsettings.json

```diff
@@ -7,7 +7,7 @@
     }
   },
   "ConnectionStrings": {
-    "OrderingDb": "Data Source=localhost,1533;Initial Catalog=CatalogueOrdering;MultipleActiveResultSets=true;User ID=NHSD;Password=DisruptTheMarket1!"
+    "OrderingDb": "Data Source=localhost,1452;Initial Catalog=CatalogueOrdering;MultipleActiveResultSets=true;User ID=NHSD;Password=DisruptTheMarket1!"

```

Comment:
Please ensure the name of the database is consistent with the cluster database name as when attempting to use the local cluster the database name we set to 'buyingcatalogue'.

Reply:
Good shout! I've changed the name as part of [this PR](https://github.com/nhs-digital-gp-it-futures/platform/pull/119). I'm planning merge them at the same time to not induce issues with the local cluster setup :)

Reply:
👍